src/main/java/com/google/finapp/FinAppService.java

```diff
@@ -16,7 +16,6 @@
 
 import com.google.inject.Inject;
 import com.google.protobuf.Empty;
-import com.google.finapp.SpannerDaoException;
```

Comment:
how come you deleted this from this file?

Reply:
The code compiled without it (and IntelliJ highlighted it as an unnecessary import). I believe since this exception is created in the same package, it doesn't need to be imported.